packages/player-react/src/index.tsx

```diff
@@ -114,6 +129,45 @@ export function Player({
     }
   };
 
+  const handlePlayerReady = (event: Event) => {
+    const player = (event as CustomEvent).detail;
+    if (player) {
+      onPlayerReady(player);
+    }
+  };
+
+  const handlePlayerResize = useCallback(
+    (entries: ResizeObserverEntry[]) => {
+      const [entry] = entries;
+      if (entry && wrapperRef.current) {
+        const newRect = entry.contentRect;
+        if (
+          !lastRect.current ||
+          newRect.width !== lastRect.current.width ||
+          newRect.height !== lastRect.current.height
+        ) {
+          console.log('jo resize');
+          lastRect.current = newRect;
+          onPlayerResize?.(newRect);
+        }
+      }
+    },
+    [onPlayerResize],
+  );
+
+  const lastRect = useRef<DOMRectReadOnly | null>(null);
```

Comment:
Since we're accessing lastRect in the handlePlayerResize function above we should put this further up just to make it less confusing. Maybe just put it with the rest of the component state.